### DISCUSSION TITLE
client desktop disappear after logout

### DISCUSSION THREAD

**Discussion by hightvibe** [epoptes/epoptes#29] [Q&A]
Hi,
I've installed the latest epoptes on my lab PCs running xubuntu 16.10 (epoptes-client on student's PCs, epoptes on teacher's one).
When a student logouts, the desktop shown in the teacher's epoptes window disappears.
I can see it again only when the student re-login
I' like to manage the client login from teacher Pc

Anyone can give me some help?

Thank you

**Comment by alkisg** [comment]
https://answers.launchpad.net/epoptes/+question/644756
